guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,242 @@
+   retina = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   alt-text = "Surreal image featuring two people extending their arms from opposite sides, exchanging dollar bills in front of a retro computer. The computer, with a blank screen and an old keyboard, sits on a small desk. One person is dressed in a pink suit, while the other wears a yellow outfit, set against a vibrant pink and orange background."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+### Creating and redeeming ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "Diagram illustrating the creation and redemption of ecash by Alice. On the left side, under ‘Creation,’ Alice sends 100 sats to an ecash mint, which creates ecash tokens, and Alice receives 100 sats in ecash form. On the right side, under ‘Redemption,’ Alice sends 100 sats in ecash form to the mint, which burns the ecash tokens, and Alice receives 100 sats back in Bitcoin. The process flow is represented with dashed arrows and labeled steps."
+   caption = "Alice mints 100 sats into ecash through a mint and later redeems the ecash tokens to reclaim her original 100 sats in bitcoin."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+Cashu and Fedimint use various terms for the processes of creating, redeeming, and transferring bitcoin-backed ecash. To provide clarity, we'll use neutral terms in our explanations and provide a translation table for specific implementations.
+
+#### Creating ecash
+
+**1. User deposits funds** - The user deposits bitcoin into a mint or federation. Typically this is done by generating a lightning invoice through the mint and paying it.
+
+**2. Ecash creation** - Upon successful payment, the wallet generates secrets and [blinds](https://en.wikipedia.org/wiki/Blinding_(cryptography)) them. The wallet then sends the blinded messages to the mint or federation, which returns [blind signature](https://en.wikipedia.org/wiki/Blind_signature) on the blinded messages. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
+
+**3. Ecash receipt** - The user receives a set of proofs that correspond to specific amounts of bitcoin. These proofs can be combined to create an ecash token of any amount denominated in bitcoin.
+
+#### Converting ecash to bitcoin
+
+**1. User initiates conversion** - User initiates a conversion request with their mint or federation.
```

Comment:
Could be helpful to mention that this happens automatically, if the user pays a Lightning invoice. It's not really a conscious step that users need to think about.